pkg/ignore/ignore.go

```diff
@@ -0,0 +1,127 @@
+package ignore
+
+import (
+	"bufio"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/openshift/source-to-image/pkg/api"
+)
+
+type DockerIgnorer struct {}
```

Comment:
let's leave it, if we come back and add .dockerignore support we'd probably want to refactor this in such a way that it takes the input file as an argument/config value rather than duplicating the whole thing anyway (since the logic would be the same)